### DISCUSSION TITLE
How to find a node in all opened maps?

### DISCUSSION THREAD

**Discussion by jabbalaci** [freeplane/freeplane#776] [Help]
I have several maps open but when I filter something, it works on the current map only. How to extend the search on all open maps?

**Comment by steveatmaat** [comment]
In the menu:
Edit->Find->Find in all open maps.

**Comment by jabbalaci** [comment]
Thanks, but I don't have that menu point (see this screenshot: https://i.imgur.com/CAuBy1h.png ). I'm using v1.10.4 . Find -> Find does the search in the current map.

**Reply by steveatmaat** [reply]
Sorry, I suggestion was incomplete. When you click on the Find menu option, a sub-menu shows several options. The one you want is "Find and replace in all maps." I know this sounds like it is only for replacing, but it is not. It will provide a popup that allows an incremental search across all open maps. You can just leave the Replace field empty.
I hope that is clearer.

**Comment by jabbalaci** [comment]
Thank you! It works now. This is what I needed. Solved.
